src/test/java/bio/terra/service/resourcemanagement/google/BucketResourceTest.java

```diff
@@ -127,22 +131,45 @@ public void createAndDeleteBucketTest() throws Exception {
         checkBucketDeleted(bucketResource.getName(), bucketResource.getResourceId());
     }
 
+    @Test
+    // create buckets in different regions and see if they're actually created there.
+    public void createBucketsInDifferentRegionsTest() throws Exception {
+        for (GoogleRegion region : Arrays.asList(GoogleRegion.US_CENTRAL1, GoogleRegion.US_EAST1)) {
+            String bucketName = "testbucket_bucketregionstest_" + region;
+            String flightId = "bucketRegionsTest_" + region;
+
+            // create the bucket and metadata
+            GoogleBucketResource bucketResource =
+                createBucket(bucketName, projectResource, region, flightId);
+
+            // Get the Bucket
+            Bucket cloudBucket = bucketService.getCloudBucket(bucketName);
+
+            assertEquals("Google bucket was created in " + region, region.toString().toLowerCase(),
+                cloudBucket.getLocation().toLowerCase());
```

Comment:
If you use `assertThat` you can use `equalToIgnoringCase()`.
```suggestion
            assertThat("Google bucket was created in " + region, cloudBucket.getLocation(), 
                equalToIgnoringCase(region.toString()));
```

Reply:
Oh cool. I'm not as familiar with the Hamcrest matchers.